docs/reference/search/explain.asciidoc

```diff
@@ -1,12 +1,9 @@
 [[search-explain]]
 == Explain API
 
-The explain api computes a score explanation for a query and a specific
-document. This can give useful feedback whether a document matches or
-didn't match a specific query.
+The explain api computes a score explanation for a query and a specific document, letting you know whether a document matches or doesn't match a specific query.
 
```

Comment:
api -> API

This is somewhat misleading/overly simplistic. Explain calculates the relevance score for a particular document and shows what led to that score. It's not just whether or not the doc matches the query. The idea is to help you understand why a doc is being scored the way it is (and why it might or might not be surfaced in results the way you expect).

For context, see https://www.elastic.co/guide/en/elasticsearch/guide/current/relevance-intro.html#explain

I'd probably change this to something like:

> The explain API calculates the relevance score for a particular document and shows what led to that score. This can help you understand why a document is (or isn't) showing up in the search results for a particular query.